python/lsst/ts/LinearStage/simulator.py

```diff
@@ -0,0 +1,28 @@
+import os
```

Comment:
Umm so there is no mention of the simulator in the documentation... but here it is! So should add some words there. Even if it just says it's very primitive